xbmc/windowing/win10/WinSystemWin10DX.cpp

```diff
@@ -152,3 +153,47 @@ void CWinSystemWin10DX::UninitHooks()
 void CWinSystemWin10DX::InitHooks(IDXGIOutput* pOutput)
 {
 }
+
+bool CWinSystemWin10DX::SetHDR(const VideoPicture* videoPicture /*not used*/)
+{
+  bool success = CWIN32Util::ToggleWindowsHDR();
+
+  if (success)
+  {
+    Sleep(2000); //Display is switching
```

Comment:
do you think it would make sense here to use the user configurable "refresh rate switch delay" here instead of the fixed 2 seconds? I suppose that HDR switching is basically taking as long as refresh rate switching, but that's just a guess.